lib/execute-method-map.ts

```diff
@@ -548,4 +548,11 @@ export const executeMethodMap = {
     command: 'background',
     params: {optional: ['seconds']},
   },
+  'mobile: simctl': {
+    command: 'mobileSimctl',
+    params: {
+      required: ['command'],
+      optional: ['args', 'timeout'],
```

Comment:
note: the timeout will work after https://github.com/appium/node-simctl/pull/254